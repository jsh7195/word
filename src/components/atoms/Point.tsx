import React from "react";
import {PointType} from '@lib/data';

const Point = (props: PointType):JSX.Element => {
  const { point } = props;
  return <span>점수 : {point ? point : 0} 점</span>;
};

export default Point;
