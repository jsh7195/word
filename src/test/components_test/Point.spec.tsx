import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Point from '../../components/atoms/Point';

Enzyme.configure({ adapter: new Adapter() });
 
test('Point Render Test ... <Point/>', () => {
  const props = {
    point: 5
  } 
  const component = renderer.create(<Point {...props}/>).toJSON();
  expect(component).toMatchSnapshot();
});