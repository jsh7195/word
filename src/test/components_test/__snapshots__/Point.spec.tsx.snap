// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Point Render Test ... <Point/> 1`] = `
<span>
  점수 : 
  5
   점
</span>
`;
